packages/playground/remote/src/lib/playground-mu-plugin/playground-includes/playground_logger.php

```diff
@@ -0,0 +1,9 @@
+<?php
+// Configure error logging
```

Comment:
`mu-plugin` is a good initial location for this, but eventually let's switch to using `BasePHP` API calls when that's supported. That way it will also impact errors thrown before this mu-plugin is included.

Reply:
>  let's switch to using BasePHP API calls when that's supported. 

What's missing today?
We can create a PHP.ini file for most settings and define the `ERROR_LOG_FILE`.
I'm just not sure about `error_reporting`.

Reply:
This should work for error reporting when defined in php.ini before the script starts:

```ini
error_reporting = E_ALL
```

What's missing is stable php.ini support for the web SAPI, I think the php.ini functions we have only reliably work with the CLI SAPI, e.g. when the `main()` functionis called.

Reply:
I started a new issue to track it https://github.com/WordPress/wordpress-playground/issues/1049